collection.go

```diff
@@ -61,13 +68,25 @@ func (c *Collection) StateCount(state State) int {
 
 	var n int
 	for _, service := range c.services {
-		if s, _ := service.GetState(); s == state {
+		if service.GetState() == state {
 			n++
 		}
 	}
 	return n
 }
 
+// UpDown returns two slices, one containing the labels of monitored services
```

Comment:
( minor minor ) comment seems not up-to-date

Reply:
Oops. 😞